openpype/hosts/maya/plugins/publish/extract_import_reference.py

```diff
@@ -0,0 +1,124 @@
+import os
+
+from maya import cmds
+
+import pyblish.api
+
+from openpype.pipeline import publish, legacy_io
+from openpype.settings import get_project_settings
+from openpype.hosts.maya.api import lib
+
+
+def _get_project_setting():
+    project_name = legacy_io.active_project()
+    project_setting = get_project_settings(project_name)
+    maya_enabled = (
+        project_setting["maya"]["publish"]["ImportReference"]["enabled"]
+    )
```

Comment:
Checking this should be redundant since if the setting is disabled the plug-in shouldn't show to begin with. Right? It should automatically apply that setting if I'm not mistaken.

Reply:
I will set the enable option of import reference into the submit maya deadline instead of creating two enabled options